navigation/eklavya_navigation/config/costmap_common_params.yaml

```diff
@@ -13,12 +13,12 @@ obstacle_layer:
    enabled: true
```

Comment:
@parthamishra1996 This is not related to simulation. You have used Abinash's commit which has changes in other folders.